### DISCUSSION TITLE
Monitor controls seem to max at 100 nits

### DISCUSSION THREAD

**Discussion by powerllama** [MonitorControl/MonitorControl#869] [Q&A] [answered]
Hello there, cool project. For some reason on my monitor when I try to change the brightness, it will only change my backlight to a max of 100 nits. Normally during the day I have my monitor set to 400 nits. Do you think this is a bug or a limitation of my monitor? Thanks

Running 2020 M1 Mac Mini using a USB-C > DisplayPort adapter.
<img width="696" alt="image" src="https://user-images.githubusercontent.com/730579/147865952-eefdaf66-a103-4345-b301-1cc50aa33b17.png">
![image](https://user-images.githubusercontent.com/730579/147865972-e9bc4dd7-2303-4588-b0c7-210f9e002171.jpeg)

**Comment by waydabber** [comment, marked as answer]
It might be a limitation of the display. You can try using m1ddc or ddctl and experiemnt with various brightness values. You can also change the max. ddc brightness level under Displays/DDC max (you need to enable `Show advanced settings`. A prudent max value to try would be something between the 100-255 range (the default max value is 100, most displays use this, but there are some exceptions).

**Reply by powerllama** [reply]
Changing the DDC Max in the advanced settings did the trick, thanks! Never thought an app like this was possible before.

**Reply by waydabber** [reply]
Great, glad it worked out!
